actions/templates/actions/actions.html

```diff
@@ -101,6 +108,12 @@ <h3><b>#{{ action.id }}</b> {{ action.name }}</h3>
     {% if action.get_log_url %}
     <p><a class="text-decoration-none text-center spacing-hover-short hvr-icon-bounce" href="{{ action.log.url }}"><i class="fas fa-file-download icon-menu hvr-icon"></i> Download log</a></p>
     {% endif %}
+
+    {% if action.status == 2 or action.status == 3 %}
```

Comment:
Can you explain what is status 2 and 3?

Reply:
The status 2 mean the actions is complete and status 3 is status is failed.